ui/router.go

```diff
@@ -58,8 +58,8 @@ func initRouters() {
 	//API:
 	beego.Router("/api/search", &api.SearchAPI{})
 	beego.Router("/api/projects/:pid([0-9]+)/members/?:mid", &api.ProjectMemberAPI{})
-	beego.Router("/api/projects/", &api.ProjectAPI{}, "get:List")
-	beego.Router("/api/projects/?:id", &api.ProjectAPI{})
+	beego.Router("/api/projects/", &api.ProjectAPI{}, "get:List;post:Post")
```

Comment:
post request without parameter 'id'.
In previous router definition, post request will route by line 'beego.Router("/api/projects/?:id", &api.ProjectAPI{})'.
If remove '?', post request will not route by that line again.
Add 'post:Post' on 'beego.Router("/api/projects/", &api.ProjectAPI{}, "get:List")' will help route post request.